app/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,9 @@
     <string name="image_track">image_track</string>
     <string name="image_more_info">moreinfo</string>
 
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.hello_blank_fragment` appears to be unused
 Unused resources make applications larger and slow down builds.